.idea/digests.iml

```diff
@@ -6,7 +6,7 @@
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
   <component name="TestRunnerService">
-    <option name="projectConfiguration" value="py.test" />
-    <option name="PROJECT_TEST_RUNNER" value="py.test" />
+    <option name="projectConfiguration" value="pytest" />
+    <option name="PROJECT_TEST_RUNNER" value="pytest" />
```

Comment:
this folder should probably be added to `.gitignore`?

Reply:
😞

Reply:
`.idea/` should be in your global `.gitignore`.

Reply:
Thanks, It was...in the process of removing myself from this mac I wiped some things I did not want to remove yet.